src/content/api/compiler-hooks.md

```diff
@@ -248,6 +248,13 @@ Called after emitting assets to output directory.
 
 - Callback Parameters: `compilation`
 
+### `assetEmitted`
+
+`AsyncSeriesHook`
+
+Allows to get byte content of emitted asset. Available since webpack v4.39.0
```

Comment:
" to get byte content" not sure what does this mean

Reply:
This is all i have, its exact bytes representing file content: https://github.com/webpack/webpack/pull/9318/files#diff-acf28c9ee14f37453776e9feeb6d7b33R9

Reply:
Its the raw data in buffer

Reply:
@montogeek if you missed notification